.circleci/config.yml

```diff
@@ -51,9 +59,33 @@ jobs:
             - run:
                   name: Publish @pxblue/icons-svg
                   command: cd svg && yarn publish:package -b $CIRCLE_BRANCH
+            - run:
+                name: Publish @pxblue/icons-mui
+                command: cd mui && yarn publish:package -b $CIRCLE_BRANCH
             - run:
                   name: Publish @pxblue/react-native-vector-icons
                   command: cd rn-vector && yarn publish:package -b $CIRCLE_BRANCH
+
+    tag:
+      docker:
+        - image: circleci/node:14.9-browsers
+      steps:
+        - checkout
+        - attach_workspace:
+            at: .
+        - gh/setup
+        - run:
+            name: Publish @pxblue/icons
+            command: yarn publish:package -b $CIRCLE_BRANCH
```

Comment:
This does a publish instead of a tag

Reply:
The other three descriptions say 'publish' but are performing a tag.

Reply:
Ah good catch, these are updated and fixed